src/main/java/Ui.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Handles printing of messages.
+ */
+public class Ui {
+    /**
+     * Prints an error message with the associated formatting.
+     * @param msg The message to be printed.
+     */
+    public static void errorMsg(String msg) {
```

Comment:
Perhaps change the method name to "printErrorMsg"?

Reply:
Perhaps change the method name to "printErrorMessage" or "printErrorMsg" as mentioned by Zhi Yuan.